source/templates/home.html

```diff
@@ -17,7 +17,13 @@
     <p class="home-download-list">
       {{hb 'home.download_links'}}
     </p>
-    <p>{{link 'pages.changelog'}}</p>
```

Comment:
You did it correctly, I think you just didn't run another `gulp build` after. When I tried locally with this branch I got the translation on releases.html as expected.